specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-11-02-preview/managedClusters.json

```diff
@@ -5091,6 +5091,71 @@
       "title": "A time range.",
       "description": "For example, between 2021-05-25T13:00:00Z and 2021-05-25T14:00:00Z."
     },
+    "ScheduledEvent": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of a scheduled event."
+        },
+        "eventid": {
+          "type": "string",
+          "description": "The event id of a scheduled event."
+        },
+        "eventsource": {
+          "type": "string",
+          "description": "The source of a scheduled event."
+        },
+        "eventstatus": {
+          "type": "string",
+          "enum": [
+            "Scheduled",
+            "Started",
+            "Completed",
+            "Cancelled"
+          ],
+          "x-ms-enum": {
+            "name": "eventstatus",
+            "modelAsString": true
+          },
+          "description": "The status of a scheduled event."
+        },
+        "scheduledtime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of a scheduled event."
+        },
```

Comment:
so does startTime and lastUpdateTime. We need to tell cx when did upgrade failed.

Reply:
added.